l10n_th_remove_tax_cash_basis/models/account_move.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2019 Ecosoft Co., Ltd (http://ecosoft.co.th/)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html)
+from odoo import models
+
+
+class AccountPartialReconcile(models.Model):
+    _inherit = 'account.partial.reconcile'
+
+    def create_tax_cash_basis_entry(self, percentage_before_rec):
```

Comment:
@saran, I now think we should not create this module, but just move this function into l10n_th_vendor_tax_invoice.
Then, add in description that the module will be deleting some move line. I think it suffice.